docs/release-notes/platform_10-9-0_releasenotes.md

```diff
@@ -0,0 +1,83 @@
+---
+id: v10.9.0
+title: Version 10.9.0 Release Notes
+sidebar_label: v10.9.0
+image: "img/release-note-link-preview.png"
+---
+
+_May 04, 2023_
+
+## Console
+
+### New Features
+
+#### Managing Service Accounts at Console level
+
+It is possible to create [Company-independent service accounts](TO-DO link) in case you need to give them privileges to perform [root-level operations](/development_suite/identity-and-access-management/console-levels-and-permission-management.md).
+
+:::info
+Only Console Super Users can [add](TO-DO link) or [delete](TO-DO link) Company independent service accounts, manageable exclusively at back-office level.
+:::
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* 
+
+## Fast Data
+
+### New Features
+
+Support to .sql file type in the DDL modal import
```

Comment:
@giuliowaitforitdavide let's rephrase it better, giving a more exhaustive explanation

Reply:
Changed